src/attached_probe.cpp

```diff
@@ -935,11 +935,15 @@ void AttachedProbe::attach_kprobe(bool safe_mode)
     return;
   }
 
+  std::string funcname = probe_.attach_point;
+  if (probe_.path.length() > 0)
+    funcname = probe_.path + ":" + funcname;
+
   resolve_offset_kprobe(safe_mode);
   int perf_event_fd = bpf_attach_kprobe(progfd_,
                                         attachtype(probe_.type),
                                         eventname().c_str(),
-                                        probe_.attach_point.c_str(),
+                                        funcname.c_str(),
```

Comment:
I don't think that this is going to work. BCC expects a kprobe name here and likely won't handle `module:function` syntax.

Reply:
IIUC, the kernel interface [1] accepts the name with the [MOD:] prefix. And I tested this patch it works. Thanks!

[1] https://www.kernel.org/doc/Documentation/trace/kprobetrace.rst

Reply:
Nice! Would you be able to finish the PR, then? It's just missing a couple of things, mainly documentation and tests.